lib/bolt/config.rb

```diff
@@ -223,7 +223,7 @@ def update_transports(data)
           if @future
             to_expand = %w[private-key cacert token-file] & selected.keys
             to_expand.each do |opt|
-              selected[opt] = File.expand_path(selected[opt], @boltdir.path) if opt.is_a?(String)
+              selected[opt] = File.expand_path(selected[opt], @boltdir.path) if selected[opt].is_a?(String)
             end
```

Comment:
Is it possible for opt not to be a string? I couldn't tell if this was a typo or if there is actually a case where it is not a string.

Reply:
I don't think so, because those are the config keys not the values. It looks like this was introduced in https://github.com/puppetlabs/bolt/commit/871e92d52dca54eedb3d5f87900cc054dcb9f7fd and is definitely a bug. Thanks Cas!